semantic-conventions/src/opentelemetry/semconv/model/semantic_convention.py

```diff
@@ -131,7 +132,12 @@ def _get_attributes(self, templates: Optional[bool]):
                 if templates is None
                 or templates == AttributeType.is_template_type(attr.attr_type)
             ],
-            key=lambda attr: attr.fqn,
+            key=lambda attr: (
+                attr.requirement_level.value
+                if attr.requirement_level
+                else RequirementLevel.OPT_IN.value,
```

Comment:
I believe `recommended` is a default requirement level

Reply:
Why python is so weird 😆. What about writing this condition in another function? I'm not good at python and this took me a while to figure it out what it is doing :D. Can ignore if this is the python way of doing it. 

```
if attr.requirement_level:
    return attr.requirement_level.value, attr.fqn
return RequirementLevel.RECOMMENDED.value, attr.fqn
```

Reply:
1. Switched the default requirement level to `RECOMMENDED`.
2. Extracted the logic for determining the comparison key to a separate function.